docker/pypi/dmwm-base/manage

```diff
@@ -9,17 +9,35 @@
 ##H   start       (re)start the service
 ##H   stop        stop the service
 
-srv=`echo $USER | sed -e "s,_,,g" | sed -e "s,t0req,t0_req,g"`
+# common settings to prettify output
 echo_e=-e
-LOGDIR=/data/srv/logs/$srv
-CFGFILE=/data/srv/current/config/$srv/config.py
-STATEDIR=/data/srv/state/$srv
 COLOR_OK="\\033[0;32m"
 COLOR_WARN="\\033[0;31m"
 COLOR_NORMAL="\\033[0;39m"
+
+# service settings
+srv=`echo $USER | sed -e "s,_,,g" | sed -e "s,t0req,t0_req,g"`
+LOGDIR=/data/srv/logs/$srv
+AUTHDIR=/data/srv/current/auth/$srv
+STATEDIR=/data/srv/state/$srv
+CFGDIR=/data/srv/current/config/$srv
+CFGFILE=$CFGDIR/config.py
+# some MS services uses different config naming convention, therefore we'll
+# adjust CFGFILE assingment
+for c in monitor output ruleCleaner transferor unmerged; do
+    if [ -f $CFGDIR/config-${c}.py ]; then
+        CFGFILE=$CFGDIR/config-${c}.py
+    fi
+done
+
+# necessary env settings for all WM services
 export PYTHONPATH=$PYTHONPATH:/etc/secrets:/data/srv/current/config/$srv
-export X509_USER_CERT=/data/srv/state/$srv/proxy/proxy.cert
-export X509_USER_KEY=/data/srv/state/$srv/proxy/proxy.key
+export X509_USER_KEY=$AUTHDIR/dmwm-service-key.pem
+export X509_USER_CERT=$AUTHDIR/dmwm-service-cert.pem
+export REQMGR_CACHE_DIR=$STATEDIR
+export WMCORE_CACHE_DIR=$STATEDIR
+export WMAGENT_CONFIG=$CFGFILE
```

Comment:
I was going to say that this WMAGENT_CONFIG variable is not necessary, but I see it has been defined in the workqueue manage script. So we better leave it around for now.

Reply:
If you want I can put if/then/fi block around it and make it available only for workqueue service. Please let me know.

Reply:
Having a second look at it, this is no longer necessary. It looks like we had this code in place because we were - at some point in the long past - pushing couchapps from the workqueue manage script. This is long not used and I would suggest to remove WMAGENT_CONFIG env var from this script.